src/bytes.rs

```diff
@@ -63,6 +79,16 @@ impl ByteBuf {
             bytes: Vec::with_capacity(cap)
         }
     }
+
+    pub fn as_vec(self) -> Vec<u8> {
+        self.bytes
+    }
+}
```

Comment:
Well, the message actually mentions `From` implementations, not `Into`.

Anyway, committed this change.